specs/config/readme.html

```diff
@@ -1,22 +1,29 @@
 <!DOCTYPE html>
-<html xmlns='http://www.w3.org/1999/xhtml'>
-<head>
-<meta charset='utf-8' />
-<title>OSLC Configuration Management Read Me</title>
-</head>
-<body>
-<p>
-  This directory contains the source files for the OSLC Configuration Management Specification,
-  which is one of the work products of the OASIS OSLC Core Technical Committee.
-</p>
-<p>
-  Earlier draft revisions of this specification may be found
-  <a href="https://tools.oasis-open.org/version-control/browse/wsvn/oslc-core/trunk/specs/config/" target="_blank">here</a>.
-</p>
-<p>
-  Even earlier draft revisions of this specification were products of the now-closed
-  Change and Configuration Management Technical Committee. Those previous drafts may be found
-  <a href="https://tools.oasis-open.org/version-control/browse/wsvn/oslc-ccm/tags/final/specs/config-mgt/" target="_blank">here</a>.
-</p>
-</body>
+<html xmlns="http://www.w3.org/1999/xhtml">
+  <head>
+    <meta charset="utf-8" />
+    <title>OSLC Configuration Management Read Me</title>
+  </head>
+  <body>
+    <p>
+      This directory contains the source files for the OSLC Configuration Management Specification, which is one of the
```

Comment:
Do you want this to be pubslieshed on OASIS doc archive?

Reply:
No, that was for internal use. There's really no need for it to be HTML - Markdown would probably be a better choice now.